adapters/conversant/conversanttest/supplemental/test_params.json

```diff
@@ -0,0 +1,258 @@
+{
+  "mockBidRequest": {
+    "id": "testauction",
+    "imp": [
+      {
+        "id": "1",
+        "bidfloor": 7,
+        "tagid": "mytag",
+        "secure": 1,
+        "video": {
+          "format": [{"w": 300, "h": 250}],
+          "pos": 1,
+          "mimes": ["video/mp4"],
+          "api": [1,2],
+          "protocols": [2],
+          "maxduration": 60
+        },
+        "ext": {
+          "bidder": {
+            "site_id": "108060"
```

Comment:
Can we please add a test case were ext-defined `bidFloor` is zero and the `bidFloor` defined at the `imp` level is negative?
```
{
  "mockBidRequest": {
    "id": "testauction",
    "imp": [
      {
        "id": "1",
        "banner": {
          "format": [{"w": 300, "h": 250}]
        },
        "bidfloor": -1,
        "ext": {
          "bidder": {
            "site_id": "108060",
          }
        }
      }
    ],
```